pkg/cli/log_flags_test.go

```diff
@@ -259,6 +377,11 @@ func TestLogFlagCombinations(t *testing.T) {
 			t.Errorf("%d. cliCtx.logConfigInput.s expected '%s', but got '%s'. td.args was '%#v'.",
 				i, td.expectedLogCfg, cliCtx.logConfigInput.s, td.args)
 		}
+
+		if !reflect.DeepEqual(td.expectedLogVars, cliCtx.logConfigVars) {
```

Comment:
don't want to change this diff since it's a backport but in hindsight, i think we could have used `require.ElementsMatch` instead of importing `reflect` which would be nice to avoid.

Reply:
Ahh, good to know. Thanks @dhartunian.